index.js

```diff
@@ -0,0 +1,25 @@
+const calculate = (arr) => {
+    const operands = ['+', '-', '/', '*']
+    let newArr = []
+
+    for (let i = 0; i < arr.length; i++) {
+        if (((!isNaN(arr[i]) && arr[i] !== '' && arr[i] !== ' ') || operands.includes(arr[i]))) newArr.push(arr[i])
+    }
+    if (newArr.length !== 3 || !operands.includes(newArr[1])) return NaN
+
+    switch (newArr[1]) {
+        case '+':
+            return Number(newArr[0]) + Number(newArr[2])
+        case '-':
+            return Number(newArr[0]) - Number(newArr[2])
+        case '*':
+            return Number(newArr[0]) * Number(newArr[2])
+        case '/':
+            return Number(newArr[0]) / Number(newArr[2])
+
+    }
```

Comment:
Switches should always have a default case. Which would eliminate the need for the `!operands.includes(newArr[1])` check on line 8.